pkg/policyengine/connectivitypdp/connectivity_pdp.go

```diff
@@ -35,22 +36,23 @@ const (
 // WorkloadAttrs are the actual key-value attributes attached to any given workload.
 type WorkloadAttrs map[string]string
 
-// PDP is the main object to maintain a set of connectivity policies and decide
+// PDP is the main object to maintain a set of access policies and decide
 // whether a given connection is allowed or denied by these policies.
 type PDP struct {
 	privilegedPolicies policyTier
 	regularPolicies    policyTier
 }
 
-// policyTier holds a set of ConnectivityPolicies, split into deny policies and allow policies
+// policyTier holds a set of AccessPolicies, split into deny policies and allow policies
 // Within a tier, no two policies can have the same name, even if one is deny and the other is allow.
 type policyTier struct {
```

Comment:
won't you need a different structure to hold the Privileged and Regular policies since they're now different structures?

Reply:
That's why I only hold policy specs, which are the same struct for AccessPolicy and for PrivilegedAccessPolicy